barretenberg/cpp/src/barretenberg/plonk/composer/standard_composer.cpp

```diff
@@ -25,7 +25,7 @@ namespace bb::plonk {
  *
  * @return Pointer to the initialized proving key updated with selector polynomials.
  * */
-std::shared_ptr<plonk::proving_key> StandardComposer::compute_proving_key(const CircuitBuilder& circuit_constructor)
+std::shared_ptr<plonk::proving_key> StandardComposer::compute_proving_key(CircuitBuilder& circuit_constructor)
```

Comment:
Had to ditch these consts since the shared trace construction logic involves updating actual circuit members. (This was already the case for ultra/goblinultra).